.github/workflows/deployToStaging.yaml

```diff
@@ -0,0 +1,48 @@
+name: deploy to Staging
```

Comment:
I guess we don't need `deployToX.yaml` files anymore, can we remove it?

Reply:
Sure. will do.